utils/install-common.sh

```diff
@@ -69,13 +87,8 @@ is_newer_version() {
 }
 
 install_apisix() {
-    # show awk version
-    awk --version
     mkdir -p /tmp/build/output/apisix/usr/bin/
     cd /apisix
-    # remove useless code for build
-    sed -i 's/url.*/url = ".\/apisix",/' rockspec/apisix-master-${iteration}.rockspec
-    sed -i 's/branch.*//' rockspec/apisix-master-${iteration}.rockspec
```

Comment:
Interesting. Is it safe to remove them?

Reply:
I test it on my local machine and check the logic, it should be safe to remove them

All the CI passed :)

Reply:
Look like we will download the APISIX from master instead of using the local directory. (Note that the CI only check the basic proxy function)

Reply:
> Look like we will download the APISIX from master instead of using the local directory. (Note that the CI only check the basic proxy function)

I will test it today, thanks for the tips.

Reply:
After test, I believe that it is safe to remove them.  :)